addons/voice/org.openhab.voice.picotts/README.md

```diff
@@ -14,6 +14,10 @@ sudo apt-get install libttspico-utils
 In Arch Linux the pico2wave binaries are available in an Arch User repository (AUR) under
 https://aur.archlinux.org/packages/svox-pico-bin/
 
+##  Availability via PaperUI
+After manual install if no options are selectable from PaperUI download the .jar file and place in addons folder.  
```

Comment:
What issue is this about and what JAR do you add to resolve this? 
If this is about adding the PicoTTS addon to an older openHAB version I wouldn't add this to the docs. 
Otherwise we would end up with such a piece of documentation for each addon. :wink:

Reply:
Issue for OH version stable 2.3.0-1 when following the documentation to install PicoTTS using apt-get install libttspico-utils. After install this .jar file  org.openhab.voice.picotts-2.4.0-SNAPSHOT.jar is needed for Pico options via PaperUI.

Reply:
openHAB 2.3.0 was released on May 28, 2018 and does not contain the Pico TTS addon. 
The Pico TTS addon was added when [PR #3125](https://github.com/openhab/openhab2-addons/pull/3125) got merged on July 27th, 2018.
So it's only part of the 2.4.0 SNAPSHOT/Milestone builds.